lib/pipelines/compileSoy.js

```diff
@@ -108,17 +108,27 @@ function extractParams() {
 			var parsed = getParsedSoy(soyJsPath, file.contents);
 
 			templateData[soyJsPath] = {};
-			parsed.templates.forEach(function(cmd) {
-				if (cmd.deltemplate) {
+
+			parsed.body.forEach(function(template) {
+				if (template.deltemplate) {
```

Comment:
I think this will always be false, since neither `Template` or `DelTemplate` have a `deltemplate` field. If the intent is to skip these, you can check if `template.type === 'DelTemplate'`.